src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/EnvelopeControllerTest.java

```diff
@@ -185,15 +174,18 @@ public void should_successfully_return_all_envelopes_with_processed_status_for_a
             // Envelope id is checked explicitly as it is dynamically generated.
             .andExpect(MockMvcResultMatchers.jsonPath("envelopes[0].id").exists());
 
-        List<Envelope> envelopesFromDb = envelopeRepository.findAll();
-
-        assertThat(envelopesFromDb.size()).isEqualTo(2);
-
-        assertThat(envelopesFromDb)
+        assertThat(envelopeRepository.findAll())
+            .hasSize(2)
             .extracting("zipFileName", "status")
-            .containsExactlyInAnyOrder(tuple("7_24-06-2018-00-00-00.zip", PROCESSED),
-                tuple("8_24-06-2018-00-00-00.zip", UPLOAD_FAILURE));
+            .containsExactlyInAnyOrder(
+                tuple("7_24-06-2018-00-00-00.zip", PROCESSED),
+                tuple("8_24-06-2018-00-00-00.zip", METADATA_FAILURE)
+            );
+
+        byte[] testPdfBytes1 = toByteArray(getResource("1111005.pdf"));
+        Pdf pdf1 = new Pdf("1111005.pdf", testPdfBytes1);
```

Comment:
Is this the pdf file from 8_24-06-2018-00-00-00.zip?
If so, it would be good to use more meaningful variable names or add a comment.